js/index.js

```diff
@@ -37,6 +37,76 @@ const siteContent = {
   },
 };
 
+// Images //
+
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
+const logo = document.getElementById("logo-img");
+const cta = document.getElementById("cta-img");
+const middle = document.getElementById("middle-img");
+
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+cta.setAttribute('src', siteContent["cta"]["img-src"])
+middle.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+// Navigation bar //
+const aTag = document.getElementsByTagName('a');
+const navData = document.querySelectorAll('nav a');
+const addA = document.createElement('a');
+const addB = document.createElement('a');
+const addToNav = document.querySelector('nav')
+
+addA.href = '#';
+addA.textContent = 'Epic'
+addToNav.appendChild(addA);
+
+addB.href = '#';
+addB.textContent = 'Win'
+addToNav.appendChild(addB);
+
+for(let i = 0; i < aTag.length; i++){
```

Comment:
Great job hitting all of the elements at once for styling. One thing to note though, if you use querySelectorAll it gives you access to forEach right out of the box, and if you use arrayFrom you get access to ALL of the array methods. Doing that will lighten the work load a little bit, and keep you from having to use an actual for loop.